qiskit_ibm_runtime/options/options.py

```diff
@@ -123,48 +124,78 @@ def _set_if_exists(name: str, _inputs: dict, _options: dict) -> None:
 
         options_copy = copy.deepcopy(options)
         sim_options = options_copy.get("simulator", {})
-        inputs: dict[str, Any] = {}
+        output_options: dict[str, Any] = {}
         coupling_map = sim_options.get("coupling_map", Unset)
         # TODO: We can just move this to json encoder
         if isinstance(coupling_map, CouplingMap):
             coupling_map = list(map(list, coupling_map.get_edges()))
-        inputs["transpilation"] = {
+        output_options["transpilation"] = {
             "optimization_level": options_copy.get("optimization_level", Unset),
             "coupling_map": coupling_map,
             "basis_gates": sim_options.get("basis_gates", Unset),
         }
 
         for fld in [
-            "resilience_level",
+            "default_precision",
+            "default_shots",
+            "seed_estimator",
+            "dynamical_decoupling",
             "resilience",
             "twirling",
-            "dynamical_decoupling",
-            "seed_estimator",
         ]:
-            _set_if_exists(fld, inputs, options_copy)
+            _set_if_exists(fld, output_options, options_copy)
+
+        resilience_options = output_options.get("resilience", {})
+        # Convert zne/pec settings to layer mitigation
+        layer_mitigation = []
```

Comment:
Please check if this should be a string or a list, because I think it needs to match Literal["zne", "pec"] | None.

(I also left another comment wrt this discrepancy)